driver/audio/SDL/src/AudioSDL.cpp

```diff
@@ -134,8 +134,20 @@ SoundHandle AudioSDL::LoadEffect(const std::string& filepath)
         return SoundHandle();
     }
 
-    SoundSDL_Effect* sd = new SoundSDL_Effect(sound);
-    return CreateSoundHandle(sd);
+    return CreateSoundHandle(new SoundSDL_Effect(sound));
+}
+
+SoundHandle AudioSDL::LoadEffect(const std::vector<char>& data, const std::string& ext)
```

Comment:
whats the ext for? I dont see where its used?

Reply:
The idea was to keep the interface generic so an implementation could fall back to writing to file and loading from there. See the original source